src/lib/format/main/index.ts

```diff
@@ -48,6 +57,30 @@ export async function formatSourceFromFile(
   log.debug('Config:', config);
   log.debug('Options:', options);
   const allConfig = await enhanceConfig(config, fileName, options);
+
+  if (fileName.endsWith('.vue')) {
+    const { descriptor } = parseVue(text);
+    const vueScript = descriptor.scriptSetup ?? descriptor.script;
```

Comment:
Is it true that there is ONLY one script in a Vue source?

Reply:
https://github.com/vuejs/vue-loader/issues/228